src/constants.ts

```diff
@@ -3,15 +3,17 @@ export const PORT = process.env.PORT || 8080
 export const ADMIN = 'admin'
 
 export const SOCKET_IO = {
+  DISCONNECT: 'disconnect',
+  DRAW_FROM_DECK: 'draw from deck',
+  DRAW_FROM_DISCARD: 'draw from discard',
+  DISCARD_FROM_HAND: 'discard from hand',
+  GO_OUT: 'go out',
   JOIN_ROOM: 'join',
   MESSAGE: 'message',
-  USER_MESSAGE: 'sendMessage',
-  ROOM_DATA: 'roomData',
+  ON_CURRENT_USER_JOIN_ROOM: 'current_user_join_room',
```

Comment:
Will eventually update the other events to use underscores instead of spaces. This doesn't affect functionality, more just readability